src/client/components/pages/Schedule/SchedulePage.tsx

```diff
@@ -46,8 +46,23 @@ export interface PrefixState{
  * A button to filter course prefixes and fade sessionblock based
  * on selected value
  */
-const PrefixButton = styled(Button)<ButtonProps & { prefix: string }>`
- background-color: ${({ prefix }) => getCatPrefixColor(prefix)};
+const PrefixButton = styled(Button) <ButtonProps & { prefix: string, isPrefixActive: (prefix: string) => boolean; }>`
+ margin: 8px;
+ color: ${({ isPrefixActive, prefix }) => (isPrefixActive(prefix) ? 'black' : `${getCatPrefixColor(prefix)}`)};
+ border: ${({ isPrefixActive, prefix }) => (isPrefixActive(prefix) ? '' : `1px solid ${getCatPrefixColor(prefix)}`)};
```

Comment:
This `border` value is causing a minor bug where the size of the overall box changes when button are clicked. I've played around with a couple of different solutions, and I think we can either:

1. Don't change the `border`, `color`, and `background-color`, but instead change the `opacity` of the button as a whole to match what's in the table.
2. Add the border to all states on the buttons (including the `:hover` and `:active` pseudo-classes

I'm personally leaning toward 1, but I think either approach is fine.